zebrad/src/components/mempool/storage.rs

```diff
@@ -78,6 +82,9 @@ pub enum SameEffectsChainRejectionError {
     #[error("best chain tip has reached transaction expiry height")]
     Expired,
 
+    #[error("transaction outpoints or nullifiers were committed to the best chain")]
+    Mined,
```

Comment:
Sometimes this happens because the transaction has been mined, but other times it happens because the output was spent by another transaction.

```suggestion
    DuplicateSpend,
```

Reply:
Renamed to DuplicateSpend in [`b5cffbf`](https://github.com/ZcashFoundation/zebra/pull/5434/commits/b5cffbfe887e5701fb6b72020c5f390e05319c90).